.travis.yml

```diff
@@ -84,6 +86,21 @@ jobs:
     - stage: Distributed Reaper Integration Tests
       name: Two Reapers on Cassandra 4.0
       env: TEST_TYPE=ccm CASSANDRA_VERSION=git:trunk GRIM_MIN=2 GRIM_MAX=2
+    - stage: Sidecar Integration Tests
+      name: Cassandra 2.1.20
+      env: TEST_TYPE=sidecar CASSANDRA_VERSION=2.1.20 GRIM_MIN=1 GRIM_MAX=1
+    - stage: Sidecar Integration Tests
+      name: Cassandra 2.2.13
+      env: TEST_TYPE=sidecar CASSANDRA_VERSION=2.2.13 GRIM_MIN=1 GRIM_MAX=1
+    - stage: Sidecar Integration Tests
+      name: Cassandra 3.0.17
+      env: TEST_TYPE=sidecar CASSANDRA_VERSION=3.0.17 GRIM_MIN=1 GRIM_MAX=1
+    - stage: Sidecar Integration Tests
+      name: Cassandra 3.11.3
+      env: TEST_TYPE=sidecar CASSANDRA_VERSION=3.11.3 GRIM_MIN=1 GRIM_MAX=1
+    - stage: Sidecar Integration Tests
+      name: Cassandra 4.0
+      env: TEST_TYPE=sidecar CASSANDRA_VERSION=git:trunk GRIM_MIN=1 GRIM_MAX=1
```

Comment:
you have to add the matching CASSANDRA_VERSION equivalents above as well.

Reply:
Done, thanks